service-api/module/Application/src/Model/Entity/CaseData.php

```diff
@@ -78,6 +78,9 @@ class CaseData implements JsonSerializable
     #[Annotation\Required(false)]
     public ?string $searchPostcode = null;
 
+    #[Annotation\Required(false)]
+    #[Annotation\Validator(Uuid::class)]
+    public string $sessionId = '7871cae1-6712-4d3f-9111-1c1e38ceb222';
```

Comment:
Can we change the name of this field to something that specifically says it's for counter service/Yoti? To avoid future confusion about what session it's referring to.

Also, this shouldn't have a default value

Reply:
Will change name. WRT default value, I needed to add this in as by the point a case entry is first created, we will not have a sessionId, but because I needed to add sessionId as an index, it always needs a value on record creation.

I could not find any work around to this yesterday, will have a think abit more what else we can do.